Osiris/Hacks/Visuals.cpp

```diff
@@ -400,23 +394,20 @@ void Visuals::skybox() noexcept
     }
 }
 
-std::wstring towstring(std::string str)
-{
-    return strconverter.from_bytes(str);
-}
-
 void Visuals::showVelocity() noexcept
 {
-    if (!config->visuals.showvelocity || !localPlayer)
+    if (!config->visuals.showvelocity.enabled || !localPlayer->isAlive())
```

Comment:
you forgot to check if !localPlayer

Reply:
Ah yes that would be why

Reply:
> Ah yes that would be why

still is crashing after enabling feature thought